src/sentry/utils/iterators.py

```diff
@@ -24,7 +28,7 @@ def shingle(n, iterator):
     )
 
 
-def chunked(iterator, size):
+def chunked(iterator: Iterable[T], size: int) -> list[T]:
```

Comment:
This returns a `list[list[T]]`

Reply:
ah indeed, this should actually be `Generator[list[T], None, None]` -- oops!